### ISSUE TITLE
✨ Feature request: Variable for number of attempts (default: 3) on install_agent role

### ISSUE THREAD

**Issue by quentin-rey** [svalabs/sva.sentinelone#37]
##### SUMMARY
It would be useful to be able to define a variable that manages the number of tries for the ‘Fail if new client does not appear in management console’ task.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
_sva.sentinelone.install_agent_

##### ADDITIONAL INFORMATION
The number of retries is defined in the `mail.yml` of the [_install_agent_ role](https://github.com/svalabs/sva.sentinelone/blob/main/roles/install_agent/tasks/main.yml), on the last task.
```yaml
- name: "Fail if new client does not appear in management console"
  ansible.builtin.uri:
    url: "{{ api_url }}agents?siteIds={{ siteid }}&computerName={{ ansible_hostname | urlencode }}&isActive=true"
    method: GET
    return_content: true
    headers:
      Accept: application/json
      Authorization: "APIToken {{ api_token }}"
    validate_certs: true
    status_code: 200
  register: registrationstatus
  delegate_to: localhost
  no_log: "{{ hide_sensitive }}"
  until: ((registrationstatus.json.data | length) > 0) and (registrationstatus.status == 200)
  retries: 3
  delay: 20
```

**Comment by mwester117** [comment]
Hi @quentin-rey,

I added the requested feature. New release will follow in the next days. I will keep you on track

**Comment by quentin-rey** [comment]
> Hi @quentin-rey,
> 
> I added the requested feature. New release will follow in the next days. I will keep you on track

Awesome, thank you for responding so quickly! 🙏

**Comment by quentin-rey** [comment]
When will the next release be deployed? Thanks for your time!

**Comment by mwester117** [comment]
> When will the next release be deployed? Thanks for your time!

I just released some new commits to main. Tests look good so far. If testing is successfull I will release tomorrow.

**Comment by mwester117** [comment]
Hi @quentin-rey,

I just released Version 2.0.0 of the collection.
You can now use `check_console_retries` and `check_console_retry_delay` to adjust the behaviour. For more details see https://svalabs.github.io/sva.sentinelone/branch/main/collections/sva/sentinelone/install_agent_role.html

Note: If you used the download_agent with `state: info` you have to switch to the new agent_info module. See [CHANGELOG](https://github.com/svalabs/sva.sentinelone/blob/main/CHANGELOG.rst#breaking-changes-porting-guide) for more details
